SpatialGDK/Source/SpatialGDK/Public/Schema/Component.h

```diff
@@ -13,4 +13,9 @@ struct Component
 	virtual void ApplyComponentUpdate(const Worker_ComponentUpdate& Update) {}
 };
 
+struct DataComponent : Component
```

Comment:
The reason for this, is just because we have components like Worker, which do not have anything like this yet. Maybe I should just add it...

Reply:
Can you elaborate on why we need this. The intion was to kill Component now that we have the owning type ComponentData. If we want helpers to make and read specific components we should just make those individually (as what exactly we want them to do varies a lot by context.

Without the StaticComponentView we don't really use s unified component abstraction anymore.

Reply:
Hm, does ComponentData cover all use cases here? If it does, I guess it's fine to switch to it, I did not think about it at first.

Reply:
Ah, yeah, this has an issue, these components convert the schema data to an UnrealStructure. E.g. EntityAcl can be modified by just writing to the read and write acl TMaps, and then we can construct ComponentData when needed. I realize EntityACLs are going away, and that is the main use case currently, but still I think we may not be able to get rid of all of these components easily.